service-automation/config.properties

```diff
@@ -41,6 +44,7 @@ Student.url=/api/students/
 #REQUEST PAYLOADS
 ################################
 Branch.Create={"name":"$name","description":"$description","branchAddressDetails":{"addressLine1":"$addressLine1","addressLine2":"$addressLine2","addressLine3":"$addressLine3","state":"$state","city":"$city","country":"$country","pincode":"$pincode"},"emailId":"$emailId","contactNumber1":"$contactNumber1","contactNumber2":"$contactNumber2"}
+Staff.Create={"staffAllotments":[{"branchStandardSection":{"id":"$id","branch":{"id":"$id","branchAddressDetails":{"id":"$id"}},"standard":{"id":"$id","subjectDetails":[{"id":"$id"}]},"section":{"id":"$id"}},"id":null}],"firstName":"$firstName","lastName":"$lastName","residentialAddress":{"addressLine1":"$addressLine1","addressLine2":null,"state":"$state","city":"$city","country":"$country","pincode":"$pincode"},"fatherName":"$fatherName","idNumber":"$idNumber","dateOfBirth":null,"dateOfJoin":null,"staffDesignation":{"id":""},"gender":"$gender","perminantAddress":{"addressLine1":"$addressLine1","addressLine2":null,"state":"$state","city":"$city","country":"$country","pincode":"$pincode"},"primaryPhoneNumber":"","primaryEmailId":"","alternatePhoneNumber":"","secondaryEmailId":""}
```

Comment:
Still, I can see hardcoded values in the payload. Eg: Addressline2 and staff designation id is empty.